src/rebar_erlc_compiler.erl

```diff
@@ -796,3 +797,13 @@ dir_recursive(Opts, Dir, CompileOpts) when is_list(CompileOpts) ->
         undefined -> rebar_dir:recursive(Opts, Dir);
         Recursive -> Recursive
     end.
+
+valid_erl_first_conf(FileList) ->
+    case FileList of
+        [] -> ?WARN("An empty file list (~p) was provided as part of your erl_files_first directive", [FileList]);
```

Comment:
I don't know that an empty file is warning-worthy since it should be equivalent to not submitting anything. Does it break stuff?

Reply:
No this does not break stuff, will remove that. I locked on to the problem of trying to handle `""` :)